resources/views/edit_product.blade.php

```diff
@@ -0,0 +1,85 @@
+  <title>Add product</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <style>
+    * {
+      font-family: "Lato", sans-serif;
+    }
+  </style>
+</head>
+
+<body>
+  <main>
+    <div class="container my-5">
+      <div class="bg-light p-5 rounded">
+        <h2 class="fw-bold fs-2 mb-5 pb-2">Edit Product</h2>
+        <form action="{{route('products.update',$product->id)}}" method="POST" class="px-md-5" enctype="multipart/form-data">
+          @csrf
+          @method('put')
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Product Title:</label>
+            <div class="col-md-10">
+              <input type="text" placeholder="BMW" class="form-control py-2" name="title" value="{{old('title',$product->title)}}" />
+              @error('productTitle')
+              <div class="alert alert-warning">{{$message}}</div>
+              @enderror
+            </div>
+          </div>
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Price:</label>
+            <div class="col-md-10">
+              <input type="number" step="0.1" placeholder="Enter price" class="form-control py-2" name="price" value="{{old('price',$product->price)}}" />
+              @error('price')
+              <div class="alert alert-warning">{{$message}}</div>
+              @enderror
+            </div>
+          </div>
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Description:</label>
+            <div class="col-md-10">
+              <textarea name="shortDescription" id="" cols="30" rows="5" class="form-control py-2">{{old('shortDescription',$product->shortDescription)}}</textarea>
+              @error('description')
+              <div class="alert alert-warning">{{$message}}</div>
+              @enderror
+            </div>
+
+          </div>
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Image:</label>
+            <div class="col-md-10">
+              <input type="file" id="image" class="form-control" name="image"  >
+              @error('image')
+              <div class="alert alert-warning">{{$message}}</div>
+              @enderror
+
+                <img src="{{asset('assets/images/' .$product->image ) }}" alt="{{$product['title']}}"   style="max-width:150px"/>
```

Comment:
what if you are trying to edit fake generated data you will not see an image here as it is in a different directory